src/content/docs/dropins/cart/cart-functions.mdx

```diff
@@ -6,59 +6,122 @@ sidebar:
   order: 6
 ---
 
+import OptionsTable from '@components/OptionsTable.astro';
 import Aside from '@components/Aside.astro';
+import CodeInclude from '@components/CodeInclude.astro';
+import cartModel from '_includes/cartModel.ts?raw';
 
 The cart drop-in component provides API functions that allow developers to retrieve and display detailed cart information dynamically.
 
 ## addProductsToCart
 
-This function adds products to a cart. It takes `sku`, `quantity`, and `optionsUIDs` as parameters.
+The `addProductsToCart` function adds products to a cart. You must supply a `sku` and `quantity`for each product. The other parameters are specified for complex product types. The function calls the [`addProductsToCart`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/add-products/) mutation.
 
-```javascript
-import { addProductsToCart } from '@/cart/api/addProductsToCart';
-
-addProductsToCart(
-  Array<{
+```ts
+export const addProductsToCart = async (
+  items: {
     sku: string;
+    parentSku?: string;
     quantity: number;
     optionsUIDs?: string[];
-  }>
-);
+    enteredOptions?: { uid: string; value: string }[];
+  }[]
+): Promise<CartModel | null>
+```
+
+<OptionsTable
+  compact
+  options={[
+    ['Parameter', 'Type', 'Req?', 'Description'],
+    ['sku', 'string', 'Yes', 'The SKU of the product.'],
+    ['parentSku', 'string', 'No', 'For a child product, the SKU of its parent product.'],
+    ['quantity', 'number', 'Yes', 'The amount or number of an item to add.'],
+    ['optionsUIDs', 'string[]', 'No', 'The selected options for the base product, such as color or size, using the unique ID for an object.'],
+    ['enteredOptions', '{ uid: string; value: string }[]', 'No', 'An array of entered options for the base product, such as personalization text.'],
+  ]}
+/>
+
+### Returns
+
+Returns a promise that resolves to a `CartModel` object or null. The `CartModel` object has the following shape:
+
+<CodeInclude code={cartModel} lang="ts" frame="none" />
+
+### Events
+
+The event bus emits the `cart/updated` and `cart/data` events with the `CartModel` as the data payload. It also publishes add-to-cart or remove-from-cart events to the Adobe Commerce Data Layer (ACDL).
```

Comment:
Do we mean the Adobe _Client_ Data Layer?

https://experienceleague.adobe.com/developer/commerce/storefront/analytics/instrumentation/#adobe-client-data-layer-acdl